sigstore/_internal/keyring.py

```diff
@@ -65,7 +72,15 @@ def __init__(self, keys: List[bytes] = []):
         """
         self._keyring = {}
         for key_bytes in keys:
-            key = load_pem_public_key(key_bytes)
+            key = None
+
+            try:
+                key = load_pem_public_key(key_bytes)
+            except UnexpectedKeyFormatError as e:
+                raise e
+            except InvalidKeyError as e:
+                key = load_der_public_key(key_bytes)
```

Comment:
This has introduced some flexibility in the keyring API (keys can now be PEM or DER encoded, rather than just PEM), so we should document it in the `__init__`'s docstring above.